src/epub/text/aunt-agatha-makes-a-bloomer.xhtml

```diff
@@ -189,7 +189,7 @@
 			<p>“It should have that effect, sir.”</p>
 			<p>“Golly!” I said, bounding for the door.</p>
 			<hr/>
-			<p>Long before I reached Aunt Agatha’s lair I could tell that the hunt was up. Divers chappies in hotel uniform and not a few chambermaids of sorts were hanging about in the corridor, and through the panels I could hear a mixed assortment of voices, with Aunt Agatha’s topping the lot. I knocked but no one took any notice, so I trickled in. Among those present I noticed a chambermaid in hysterics, Aunt Agatha with her hair bristling, and the whiskered cove who looked like a bandit, the hotel manager fellow.</p>
```

Comment:
We shouldn't change this, divers is its own word and is no longer updated by se modernize-spelling